bgpd/bgpd.h

```diff
@@ -465,6 +465,7 @@ struct bgp {
 	/* vpn-policy */
 	struct {
 		struct ecommunity *rtlist[BGP_VPN_POLICY_DIR_MAX];
+		struct ecommunity *import_redirect_rtlist;
```

Comment:
Is this freed on exit?

Reply:
I will add some code, when calling bgp_delete().
note that the leak code seems to not freeing its context.